src/services/users.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+let users = [];
+
+const getInit = () => {
+  users = [];
+};
+
+const getAll = () => users;
+
+const getById = (userId) => {
+  const foundUser = users.find(user => user.id === userId);
+
+  return foundUser || null;
+};
+
+const create = (name) => {
+  const max = Math.max(users.map((user) => user.id)) || null;
+
+  const newUser = {
+    id: max + 1,
+    name,
+  };
+
+  users.push(newUser);
+
+  return newUser;
+};
+
+const remove = (userId) => {
+  users = users.filter(user => user.id !== userId);
+};
+
+const update = (userId, name) => {
+  const foundUser = getById(userId);
+
+  Object.assign(foundUser, { name });
```

Comment:
Same situation. maybe we can return object with new user data from this method, not just mutate?